editoast/src/models/rolling_stock/light_rolling_stock.rs

```diff
@@ -43,6 +43,8 @@ pub struct LightRollingStockModel {
     metadata: DieselJson<RollingStockMetadata>,
     power_restrictions: Option<JsonValue>,
     energy_sources: DieselJson<Vec<EnergySource>>,
+    electrical_power_startup_time: f64,
```

Comment:
I believe that we use `LightRollingStock` like `RollingStock` but only with simplified effort curves since they're the "heavy" part of the data.

For example we also have the mass and the rolling_resistance values which are equivalently useless for display